webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2611,13 +2624,60 @@ var userOverride = {
                     // callback when plugin initilized successfully.
                     MCK_ON_PLUGIN_INIT('success', data);
                 }
+                
+                // Loading zopim sdk for zendesk chat integration
+                if (kommunicate._globals.zendeskApiKey) {
+                    loadZopimSDK();
+                    var events = {
+                        'onMessageSent': handleUserMessage,
+                        'onMessageReceived': handleBotMessage,
+                    };
+                    Kommunicate.subscribeToEvents(events);
+                }
 
                 var kmChatLoginModal = document.getElementById(
                     'km-chat-login-modal'
                 );
                 kmChatLoginModal.style.visibility = 'hidden';
             };
 
+            function loadZopimSDK() {
```

Comment:
can these be moved to a separate js file so that sidebox file doesn't keep on increasing

Reply:
done